packages/hash/api/src/graphql/typeDefs/knowledge/entity.typedef.ts

```diff
@@ -17,6 +17,13 @@ export const entityTypedef = gql`
     entity: EntityDefinition!
   }
 
+  input LinkEntityMetadata {
```

Comment:
Do we want this btw? Or do we want to be consistent with the others like `EntityMetadata` and just use the scalar?

Reply:
Ah I didn't think of using scalars as inputs, much prefer this and have removed the type in https://github.com/hashintel/hash/pull/1510/commits/08dff0ed682e439cb2512b14d4910d3dbda154bf